components/card.css

```diff
@@ -0,0 +1,216 @@
+.card-collection {
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: 3rem;
+  margin-left: 5rem;
+  gap :1rem
+}
+
+.card {
+  background-color: var(--card-bg-color);
+  text-align: left;
+  padding: 0.9rem;
+  margin: 0.6rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  border-radius: 0.3rem;
+  width: 14rem;
+  height: 26rem;
+}
+
+.brand-img {
+  width: 12rem;
+  height: 12rem;
+}
+
+.card-info {
+  margin-top: 0.6rem;
+}
+.product-name {
+  color: var(--product-txt-color);
+  font-size: 1.2rem;
+}
+
+.description {
+  color: var(--card-txt-color);
+}
+
+/* rating-icon */
+.green-rate {
+  display: flex;
+  align-items: center;
+  font-size: 0.6rem;
+  width: 1.8rem;
+  height: 0.9rem;
+  background-color: var(--ratg-bg-color);
+  color: var(--card-white);
+  justify-content: center;
+  border-radius: 3px;
+  margin: 0.3rem 0;
+}
+
+.star-icon {
+  color: var(--card-white);
+  font-size: 0.4rem;
+}
+
+.price {
+  color: var(--price-txt-color);
+  display: flex;
+  margin: 0.5rem;
+}
+
+s {
+  font-size: 0.6rem;
+  margin-left: 0.3rem;
+}
+.saving {
+  margin-left: 0.4rem;
+  color: var(--card-txt-color);
+  font-size: 0.8rem;
+}
+
+.opa {
```

Comment:
You can  use something like this for this class names `.opa-1/2`

Reply:
changed to transparent though